content/docs/guides/integrate-with-apps.md

```diff
@@ -0,0 +1,199 @@
+---
+$title: Integrate AMP with your app 
+$order: 12
+$category: Develop
+toc: true
+
+---
+
+This guide is intended for app developers who want to integrate AMP pages into
+their app.
```

Comment:
Maybe explain a bit when this is useful, for example: 'This guide is intended for developers who want to link to the AMP version of content surfaced in their mobile or web app, for example, a mobile chat application could show the fast loading AMP version of a shared URL to provide better user experience.' (this still sounds a bit rough)

Reply:
Good point.  As this guide has information on AMP viewer, I hesitate saying it's just for linking. How about:

This guide is intended for mobile and web app developers who want to integrate and link to AMP pages. For example, consider a mobile chat app that loads the AMP version of a shared URL to achieve a faster experience for users.

Reply:
Way better :-)